docs/docs/libraries/esp32-lcd-library.md

```diff
@@ -7,104 +7,100 @@ tags: ["OXRS-IO-LCD-ESP32-LIB", "TAG2", "TAG3"]
 > SKU: OXRS-IO-LCD-ESP32-LIB
 
 ## Introduction
-Shoulder picanha tongue cow kevin. Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta. Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. Buffalo boudin meatball ball tip spare ribs. Venison cow pig shank pastrami, chislic ham. Beef ribs hamburger sirloin brisket, capicola drumstick cupim sausage shank burgdoggen swine.
+This library serves a common status display for OXRS compatible controller with a LCD.
 
-Cupim drumstick swine venison short loin capicola bacon picanha spare ribs tongue burgdoggen pork chop cow ham. Shank brisket ball tip tri-tip cow jerky shoulder chicken, corned beef alcatra filet mignon. Biltong jerky picanha ham hock swine. Ham hock shank corned beef, porchetta strip steak venison sirloin turducken hamburger ham rump jerky. Frankfurter pork tri-tip beef ribs meatloaf picanha pastrami doner burgdoggen.
+![Image Alt Text](/images/LCD-ScreenShots.jpg)
 
----
+The screen shots above show examples for 
+  - State Monitor     (input only, 128 inputs, switches, buttons, ...) 
+  - State Controller  (output only, 128 outpus, relais)
+  - Smoke Detector    (input and output, 16 inputs, 32 outputs, special use case smoke detector)
+
+For most of the fields the displayed information is the same for all three different I/O-boards, except the port animation is dedicated to the specific I/O.
 
-### How does it work?
-Shoulder picanha tongue cow kevin. Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta. Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. 
+The header shows the maker logo together with FW related information (details on the logo are below). 
 
-Buffalo boudin meatball ball tip spare ribs. Venison cow pig shank pastrami, chislic ham. Beef ribs hamburger sirloin brisket, capicola drumstick cupim sausage shank burgdoggen swine.
+The second block shows 
+  - the DHCP supplied IP-address. The virtual LED to the left shows the ethernet link status up/down in green/red. This information is self discovered by the lib.
```

Comment:
Might want to make this a little more generic, to save having to update if/when we support static ip? I.e. drop the `DHCP supplied` bit. Same with the `ethernet link status` - perhaps just `IP link status` in case the device is using WiFi.